Core/Config.cpp

```diff
@@ -865,6 +865,7 @@ static const ConfigSetting networkSettings[] = {
 	ConfigSetting("ForcedFirstConnect", &g_Config.bForcedFirstConnect, false, CfgFlag::PER_GAME),
 	ConfigSetting("EnableUPnP", &g_Config.bEnableUPnP, false, CfgFlag::PER_GAME),
 	ConfigSetting("UPnPUseOriginalPort", &g_Config.bUPnPUseOriginalPort, false, CfgFlag::PER_GAME),
+	ConfigSetting("UPnPTimeout", &g_Config.iUPnPTimeout, 2000, CfgFlag::PER_GAME),
```

Comment:
Hmm.. since upnp timeout usually external issue, may be not supposed to be PER_GAME setting.